pkg/cloud/vsphere/services/govmomi/create.go

```diff
@@ -56,6 +56,13 @@ const (
 
 	// nodeRole is the label assigned to every node in the cluster.
 	nodeRole = "node-role.kubernetes.io/node="
+
+	// macAddrOUI is the OUI prefix to use with generated MAC addresses
+	// Use "00:50:56" as it is VMware's OUI.
+	// The preferred OUI is "00:00:0f", the old NeXt OUI, but the in-tree
+	// cloud provider has a list of valid OUIs, and this OUI is not in the
+	// list.
+	macAddrOUI = "00:50:56"
```

Comment:
macAddrOUIPrefix

Reply:
The OUI *is* the prefix.


![](https://upload.wikimedia.org/wikipedia/commons/thumb/9/94/MAC-48_Address.svg/1280px-MAC-48_Address.svg.png)